### ISSUE TITLE
Update WFS3 via OWSLib

### ISSUE THREAD

**Issue by borrob** [geopython/GeoHealthCheck#313]
With PR #312 the OWSLib is updated to version 0.19 which seem to open possibilities to improve on the WFS probe. See TODO in the code base like [here](https://github.com/geopython/GeoHealthCheck/blob/61592f3a161b2ea66b8def6fbfb006ce9668eda8/GeoHealthCheck/plugins/probe/wfs3.py#L87) and [here](https://github.com/geopython/GeoHealthCheck/blob/61592f3a161b2ea66b8def6fbfb006ce9668eda8/GeoHealthCheck/plugins/probe/wfs3.py#L232).

Can we also enable the [WFS3Caps probe](https://github.com/geopython/GeoHealthCheck/blob/61592f3a161b2ea66b8def6fbfb006ce9668eda8/GeoHealthCheck/plugins/probe/wfs3.py#L292) and [WFS3Collection probe](https://github.com/geopython/GeoHealthCheck/blob/61592f3a161b2ea66b8def6fbfb006ce9668eda8/GeoHealthCheck/plugins/probe/wfs3.py#L331) ? Shall we rename them to `OGC API` or `OGC Feature API` ?

**Comment by tomkralidis** [comment]
Note updates https://github.com/geopython/OWSLib/pull/653

+1 to rename probes.  Perhaps `OGCAPIFeaturesLandingPage` and `OGCAPIFeaturesCollection` ?  Future probes can carry on with `OGCAPI...` pattern.  Thoughts?

**Comment by justb4** [comment]
+1 to rename Probes, but in separate issue, as is quite involved:

1) Rename Probe enum type [OGC:WFS3](https://github.com/geopython/GeoHealthCheck/blob/master/GeoHealthCheck/enums.py#L74) to I propose `OGC:OAPIF`.

2) Probe class renaming will also require (Alembic) database upgrades for existing deployments:

  * table: `resource` column `resource_type` rename typename
  * table: `probe_vars` column `probe_class` rename class-names

On demo.geohealthcheck.org database upgrades are automatically applied on auto-redeploy.

**Comment by tomkralidis** [comment]
+1.  For enum types, we can change sooner than later or wait until https://github.com/opengeospatial/oapi_common/issues/35 is addressed by OGC.

**Comment by borrob** [comment]
+1 for enums, but indeed in a separate issue. To avoid having to change it again, I would suggest waiting until OGC made it official

**Comment by justb4** [comment]
Ok, renaming within the DB is easy via Alembic, the upgrades can use SQLAlchemy like:
```
    resources = Resource.query.all()
    for resource in resources:
        if resource.resource_type == 'OGC:WFS3':
            resource.resource_type = 'OGC:OAFeat'
            print('Changed resource type for: %s' % resource.url)
            break
    DB.session.commit()
```

Similar for `probe_vars` table.

**Comment by justb4** [comment]
Interesting question: how to enforce JSON-only responses from an OAFeat endpoint compliant with standard: `f=json` or content negotiation (`Accept:` header in HTTP request). See also: https://gitter.im/opengeospatial/WFS_FES?at=5f9962bfc10273610acf016d

**Comment by justb4** [comment]
Most thus solved in #338. Closing, remaining TODO spread over two issues: #339 and #340.
